app/celery/letters_pdf_tasks.py

```diff
@@ -116,6 +116,17 @@ def update_billable_units_for_letter(self, notification_id, page_count):
         )
 
 
+@notify_celery.task(
+    bind=True, name="update-validation-failed-for-templated-letter", max_retries=15, default_retry_delay=300
+)
+def update_validation_failed_for_templated_letter(self, notification_id, page_count):
+    notification = get_notification_by_id(notification_id, _raise=True)
+    notification.status = NOTIFICATION_VALIDATION_FAILED
+    dao_update_notification(notification)
+    current_app.logger.info(f"Letter notification id: {notification_id} reference {notification.reference}: "
+                            f"validation failed: letter is too long {page_count}")
```

Comment:
hmmm can we make this a bit more readable? how does `validation failed: letter is too long ({page_count pages})` sound? just a little tweak to make it scan as a sentence